### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by MariaAfanasyeva**
Authenticated users can add comments to bots

### CODE REVIEW

File: frontend/src/api/refreshApi.js [hunk 1]
```diff
@@ -14,9 +11,15 @@ export const refresh = () => {
     mode: "cors",
     body: JSON.stringify(data),
   };
-  return fetch("http://127.0.0.1:8000/api/token/refresh/", options)
+  let tokenStatus;
+  fetch("http://127.0.0.1:8000/api/token/refresh/", options)
     .then((res) => res.json())
     .then((result) => {
-      localStorage.setItem("access_token", result.access);
+      if (result.code === "token_not_valid") {
```
Comment (artsiom-streetshares) [inline frontend/src/api/refreshApi.js:17]: hmm, should you rather check some message on the result, not status code?

Comment (artsiom-streetshares) [reply frontend/src/api/refreshApi.js:17]: or compare to some known status code

File: frontend/src/components/addComment.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    this.updateInputValue = this.updateInputValue.bind(this);
+  }
+
+  onSubmit(event) {
+    event.preventDefault();
+    const data = {
+      content: this.state.content,
+    };
+    if (this.props.botId && !this.props.commentId) {
+      const url = `http://0.0.0.0:8000/api/bot/${this.props.botId}/comment`;
```
Comment (artsiom-streetshares) [inline frontend/src/components/addComment.js]: lots of duplication between if and else. is this possible to improve?

**Review by artsiom-streetshares — APPROVED** [review]

File: frontend/src/components/botPage.js [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+                                  />
+                                </svg>
+                              </a>
+                              <a
+                                className="card-link"
+                                onClick={() =>
+                                  this.handleUpdateClick(comment.id)
+                                }
+                              >
+                                <svg
```
Comment (daniilzinevich) [inline frontend/src/components/botPage.js:315]: Try not inlining svg's manually

**Pull request merged**